rollup.config.js

```diff
@@ -67,10 +68,18 @@ const external = (id) =>
 // Pragmatically create a Rollup config for each package
 const getRollupConfig = ({ pwd, babelConfigFile }) => {
   const SOURCE_DIR = path.resolve(pwd);
+  const PRIVATE_PATH = `${SOURCE_DIR}/src/private.js`;
+
   // Get the package.json file
   const pkgConfig = require(`${SOURCE_DIR}/package.json`);
+
   // Relative input location for Rollup to bundle from
-  const input = [`${SOURCE_DIR}/src/index.js`, `${SOURCE_DIR}/src/private.js`];
+  const input = [`${SOURCE_DIR}/src/index.js`];
+
+  // If we have a private export, add it to our list of inputs
+  if (fs.existsSync(PRIVATE_PATH)) {
+    input.push(PRIVATE_PATH);
+  }
```

Comment:
After tests rolling back a few other things, this is really the only change that was needed. This rollup config should work with anything that has proper entry points defined now. The only edge case was us having a private export, which is now supported with this check. Everything else works correctly as it was (minus a discovery of an error about exports: auto in CJS)